src/Components/CriticalCareRecording/HemodynamicParameters/CriticalCare__HemodynamicParametersEditor.res

```diff
@@ -0,0 +1,137 @@
+let str = React.string
+
+let handleSubmit = (handleDone, state) => {
+  handleDone(state)
+}
+
```

Comment:
Call it by the module name since we've a common types file 
eg: don't use `CriticalCare__HemodynamicParameters.pulse` use `HemodynamicParameters.pulse` instead.